src/helpers/getIconUrlFromIssuer.ts

```diff
@@ -0,0 +1,101 @@
+import StellarSdk, { StellarTomlResolver } from "stellar-sdk";
```

Comment:
this is mostly lifted from Freighter, where we do something similar.

Ideally, we have an API do this for us. Maybe for v2!